src/main.rs

```diff
@@ -243,57 +252,53 @@ fn main() {
         }
     }
 
-    let dir = path.join(&args.name);
-    std::fs::create_dir(&dir).unwrap();
+    let project_dir = path.join(&args.name);
+    fs::create_dir(&project_dir)?;
 
     for &(file_path, contents) in template_files::TEMPLATE_FILES.iter() {
-        let path = dir.join(file_path);
+        if file_path == "Cargo.lock" {
+            continue;
+        }
```

Comment:
Is this check actually required? I don't see `Cargo.lock` listed in `TEMPLATE_FILES`

Reply:
I think this can be removed, the template/Cargo.lock is git ignored.